### ISSUE TITLE
Add additional visual distinction to messages which are visible to the current user, but cannot be seen by other users.

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#12128]
Private, gm-only, whispers, or other types of chat messages currently only have a slight change in background coloration and a different colored border. It would be good to have a non-color-based visual distinction on the chat message, such as a background watermark or corner icon (perhaps matching the roll mode).

In addition, it would be beneficial to indicate that the 'mode' buttons for chat messages only apply to messages containing a roll.

**Comment by demotech89** [comment]
I would love to see this implemented for color blind individuals. As a Gm the color differences are hard to differentiate.
